addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14912,13 +14922,13 @@ msgstr ""
 #. Description of settings category "Music -> Playback" with label #14086
 #: system/settings/settings.xml
 msgctxt "#36264"
-msgid "Category containing settings for how music playback is handled."
+msgid "This category contains the settings for how music playback is handled."
 msgstr ""
 
 #. Description of setting "Music -> Playback -> Play next song automatically" with label #489
 #: system/settings/settings.xml
 msgctxt "#36265"
-msgid "Automatically plays the next item in the current folder. For example, in Files View: After a track has been played, the next track in the same folder would automatically play."
+msgid "Automatically plays the next item in the current folder (e.g. in \"Files\" view after a track has been played, the next track in the same folder will automatically play.)"
```

Comment:
Just discussed this with @da-anda 

We're both of the opinion that braces is acceptable where an e.g. is used for a short example in the middle of an sentence, so "Some text (e.g. some text) some text." as a format is fine.

Where longer examples are constructed as a full sentence, then we would rather stick with the "For example some text" format.